modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -67,27 +66,35 @@ we officially support:
 
 [NOTE]
 ====
-If you use Ubuntu 16.04 and want to use PHP 7.x:
-
-* PHP 7.1 and 7.2 are only available via PPA. 
-  To add {ppa-guide-url}[a PPA (Personal Package Archive)] to your system, use this command: `sudo add-apt-repository ppa:user/ppa-name`.
-* PHP 7.2 standard installable, but you have to install some mandatory modules yourself, such as 
-{php-intl-ext-url}[intl].
+For Linux distributions, we support, if technically feasible, the latest two versions per platform and the previous https://wiki.ubuntu.com/LTS[LTS].
```

Comment:
Use macros for URL´s as we do in all the other documents

Reply:
sounds like an order

Reply:
Sorry, not intended to be one, just a note